dev/devicelab/manifest.yaml

```diff
@@ -32,23 +32,74 @@
 #       a custom task timeout, specified in minutes.
 
 tasks:
-  # Deviceless tests
+  # Tests of compiling in a variety of modes
 
-  # TODO: make these not require "has-android-device"; it is only there to
-  #       ensure we have the Android SDK.
+  complex_layout_android__compile:
```

Comment:
Yeah, I know. The name clash (between "build" for compilation and "build" for rendering) is causing a ton of hassle though, so I think it might be worth it.

Reply:
(I'm also happy to wait if you think that adding the ability to rename things is something we might be able to do soon, but I doubt it is. :-) )

Reply:
You don't have to wait. If we add this ability, we should be able to support moving data between two existing time series.

Reply:
Ok, cool. Let's go ahead with this and if we add support for moving data we can move it then. :-)